internal/semver/semver.go

```diff
@@ -6,24 +6,38 @@ SPDX-License-Identifier: AGPL-3.0-only
 
 /*
 Package semver provides functionality to parse and process semantic versions, as they are used in multiple components of Constellation.
+
+The official semantic versioning specification [1] disallows leading "v" prefixes.
+However, the Constellation config uses the "v" prefix for versions to make version strings more recognizable.
+This package bridges the gap between Go's semver pkg (doesn't allow "v" prefix) and the Constellation config (requires "v" prefix).
+
+[1] https://semver.org/.
```

Comment:
```suggestion
The official [semantic versioning specification] disallows leading "v" prefixes.
However, the Constellation config uses the "v" prefix for versions to make version strings more recognizable.
This package bridges the gap between Go's semver pkg (doesn't allow "v" prefix) and the Constellation config (requires "v" prefix).

[semantic versioning specification]: https://semver.org/
```
This properly embeds the link.
See the [Go doc comments page](https://tip.golang.org/doc/comment#links)